backend/README.md

```
@@ -0,0 +1,25 @@
+
+```bash
+python -m venv env
+```
+**Note:** Make sure to activate your virtual environment before installing packages:
+
+- Windows:
+```bash
+.\env\Scripts\activate
+```
+
+## Installing Dependencies
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)